config/puma.rb

```diff
@@ -23,7 +23,7 @@
 worker_timeout 3600 if ENV.fetch('RAILS_ENV', 'development') == 'development'
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
-port ENV.fetch('PORT') { 5000 }
+port ENV.fetch('PORT') { 3000 }
 
```

Comment:
what was the reason for changing this back to 3000?

Reply:
this makes it so that it defaults to 3000 if you aren't running it in the docker